<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotamHistoryBrowser</class>
 <widget class="QWidget" name="NotamHistoryBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NOTAM History list</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Filter FIR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="firFilter"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Filter type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="natureFilter"/>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="fromDateFilter">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="toDateFilter">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="applyTimeFilter">
           <property name="text">
            <string>Apply time filter</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filterEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="filterAction">
           <property name="statusTip">
            <string>Filter NOTAMs by specified text</string>
           </property>
           <property name="whatsThis">
            <string>Filter NOTAMs by specified text</string>
           </property>
           <property name="text">
            <string>Filter NOTAMs</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QtSqlTableView" name="table">
         <property name="autoScroll">
          <bool>false</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="tabKeyNavigation">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Linked NOTAMs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QtSqlTableView" name="secondTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="description"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtSqlTableView</class>
   <extends>QTableView</extends>
   <header>qtsqltableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
